R/gt.R

```diff
@@ -84,7 +84,7 @@ gt <- function(data,
                groupname_col = dplyr::group_vars(data),
                rownames_to_stub = FALSE,
                auto_align = TRUE,
-               id = random_id(),
+               id = ":random:",
```

Comment:
Change the default to `NULL`, and at render time, turn `NULL` into a random id.

Reply:
And don't accept `NA` as an id.

Reply:
Now fixed.